.github/workflows/tests.yml

```diff
@@ -6,34 +6,21 @@ on:
   pull_request:
     branches: [main]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
 jobs:
   build:
-    name: |
-      build (Python ${{ matrix.python-version }},
-      Django ${{ matrix.django-version }},
-      Wagtail ${{ matrix.wagtail-version }}),
```

Comment:
Removed so that running tests have a more meaningful sidebar title. The title will now show the Python version been test in each job.

Reply:
This looks good, it's much clearer